ts/features/zendesk/screens/ZendeskAskSeeReportsPermissions.tsx

```diff
@@ -0,0 +1,191 @@
+import EmailIcon from "../../../../img/assistance/email.svg";
+import FiscalCodeIcon from "../../../../img/assistance/fiscalCode.svg";
+import NameSurnameIcon from "../../../../img/assistance/nameSurname.svg";
+import { H1 } from "../../../components/core/typography/H1";
+import { H3 } from "../../../components/core/typography/H3";
+import { H4 } from "../../../components/core/typography/H4";
+import { H5 } from "../../../components/core/typography/H5";
+import { Link } from "../../../components/core/typography/Link";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../../components/ui/FooterWithButtons";
+import I18n from "../../../i18n";
+import {
+  AppParamsList,
+  IOStackNavigationProp,
+  IOStackNavigationRouteProps
+} from "../../../navigation/params/AppParamsList";
+import { useIOSelector } from "../../../store/hooks";
+import {
+  profileEmailSelector,
+  profileFiscalCodeSelector,
+  profileNameSurnameSelector
+} from "../../../store/reducers/profile";
+import { showToast } from "../../../utils/showToast";
+import { openWebUrl } from "../../../utils/url";
+import { ZendeskParamsList } from "../navigation/params";
+
+export type ZendeskAskSeeReportsPermissionsNavigationParams = {
+  assistanceForPayment: boolean;
+};
+
+/**
+ * id is optional since some items should recognized since they can be removed from the whole list
+ * i.e: items about profile || payment
+ */
+type Item = {
+  id?: string;
+  icon: ReactNode;
+  title: string;
+  value?: string;
+  zendeskId?: string;
+  testId: string;
+};
+
+type ItemProps = {
+  fiscalCode: string;
+  nameSurname: string;
+  email: string;
+};
+
+const iconProps = { width: 24, height: 24 };
+
+const getItems = (props: ItemProps): ReadonlyArray<Item> => [
+  {
+    id: "profileNameSurname",
+    icon: <NameSurnameIcon {...iconProps} />,
+    title: I18n.t("support.askPermissions.nameSurname"),
+    value: props.nameSurname,
+    testId: "profileNameSurname"
+  },
+  {
+    id: "profileFiscalCode",
+    icon: <FiscalCodeIcon {...iconProps} />,
+    title: I18n.t("support.askPermissions.fiscalCode"),
+    value: props.fiscalCode,
+    testId: "profileFiscalCode"
+  },
+  {
+    id: "profileEmail",
+    icon: <EmailIcon {...iconProps} />,
+    title: I18n.t("support.askPermissions.emailAddress"),
+    value: props.email,
+    testId: "profileEmail"
+  }
+];
+
+const ItemComponent = (props: Item) => (
+  <ListItem testID={props.testId}>
+    <View
+      style={{
+        flex: 1,
+        flexDirection: "row",
+        alignItems: "center"
+      }}
+    >
+      <View>{props.icon}</View>
+      <View hspacer />
+      <View style={{ flex: 1, flexDirection: "column" }}>
+        <H4>{props.title}</H4>
+        {props.value && <H5 weight={"Regular"}>{props.value}</H5>}
+      </View>
+    </View>
+  </ListItem>
+);
```

Comment:
This component is duplicated inside `ZendeskAskPermissions`. We can share it and move it inside `ts/features/zendesk/components`